### DISCUSSION TITLE
[Question][Help] How to change the name of the apk when building it?

### DISCUSSION THREAD

**Discussion by TheKnight-Dev** [flet-dev/flet#3698] [Q&A] [answered]
### Question

My project's name and also the folder name is "Simple Unit Converter", I tried to use the `--project` flag like so `--project "Simple Unit Converter"` but that didn't work because when i install it onto a device it shows simple_unit_converter, how can i fix that plz?

### ------------------------------------------------------

- [x] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
I think this issue has been fixed.
Upgrade to the latest prerelease and retry: `pip install flet --pre` (https://pypi.org/project/flet/#history)

**Reply by TheKnight-Dev** [reply]
The pre-release version didn't work😔, i did the same i did before, `flet build apk --project "Simple unit converter" --build-version 1.0.0`, but still, and also the initials of the name of the apk are all lowercase.

**Reply by ndonkoHenri** [reply]
Someone has it fixed here: https://github.com/flet-dev/flet/issues/3700#issuecomment-2247214003
What is the output of `flet --version` ?

**Reply by TheKnight-Dev** [reply]
flet --version = 0.24.0.dev3171
So all i need to do is --product instead of --project??
Anyways i'll try it right now.

EDIT: I just found out that an hour ago flet 0.24.0dev3174 was released, updating flet now.

**Comment by TheKnight-Dev** [comment, marked as answer]
--product flag worked!
